pkg/controller/installconfig/installconfig_controller.go

```diff
@@ -134,7 +145,8 @@ func (r *ReconcileInstallConfig) EnsureCheSubscription(logger logr.Logger, insta
 	} else if subCreated {
 		return subCreated, nil
 	}
-	return false, nil
+
+	return false, r.StatusUpdate(logger, installConfig, r.setStatusCheSubscriptionReady, che.SubscriptionSuccess)
 }
 
 func (r *ReconcileInstallConfig) ensureCheNamespace(logger logr.Logger, installConfig *v1alpha1.InstallConfig) (bool, error) {
```

Comment:
we can have `ns string` as param, like other methods for consistency and make explicit about the namespace needed as input.

Reply:
If we can get the namespace from InstallConfig, I don't see good reason why should we pass it as function params. So I would keep it :)